components/input/TextArea.razor

```diff
@@ -34,13 +33,13 @@
 		@if (Suffix != null)
 		{
 			<span class="@_warpperClassMapper.Class">
-				<textarea @ref="Ref" @attributes="attributes"/>
+				<textarea @ref="Ref" @attributes="attributes" @onclick:stopPropagation="@StopPropagation" @onblur:stopPropagation="@StopPropagation"/>
```

Comment:
Did you mean to do @onchange:stopPropagation=... @anddrzejb?

Reply:
Wow...amazing...Can't even copy right.... You are absolutely correct. Already posted #1927 fix.